README.md

```diff
@@ -1 +1,205 @@
-# rada.content
\ No newline at end of file
+
+# Github-Guide-Contribute-to-earn
+
+# Table of Contents
+**[1. How to contribute to Github Project](#How-to-contribute-to-Github-Project)**
+
+**[2. Hơw to contribute to Rada content](#Hơw-to-contribute-to-Rada-content)**
+
+**[3. Structure of Rada content](#Structure-of-Rada-content)**
+
+
+
+
+
+## 1. How to contribute to Github Project
+
+**1.1. Why should you contribute?**
+- Improve software, product you rely on
+- Improve existing skills
+- Meet people who are interested in similar things
+- Find mentors and teach others
+- Build public artifacts that help you grow a reputation (and a career) 
+- It’s empowering to be able to make changes, even small ones
+
+**1.2. Steps to contribute to an existing project**
+
+**-	Step 1: Fork the project repository**
+Find the project's repository on GitHub, and then "fork" it by clicking the **Fork** button in the upper right corner:
+
+<img align="middle" width="800" src="https://firstcontributions.github.io/assets/Readme/fork.png" alt="fork this repository" />
+
+**- Step 2: Clone the repository**
+
+Now clone the forked repository to your machine. Go to your GitHub account, open the forked repository, click on the code button and then click the _copy to clipboard_ icon.
+
+
+<img width="800" src="https://firstcontributions.github.io/assets/Readme/clone.png" alt="clone this repository" />
+
+
+**- Step 3: Open Pycharm/Terminal and run the following git command:**
+
+```
+git clone "url you just copied"
+```
+
+where "url you just copied" (without the quotation marks) is the url to this repository (your fork of this project). See the previous steps to obtain the url.
+
+<img width="800" src="https://firstcontributions.github.io/assets/Readme/copy-to-clipboard.png" alt="copy URL to clipboard" />
+
+For example:
+
+```aa
+git clone https://github.com/this-is-you/Github-Guide-Contribute-to-earn.git
+```
```

Comment:
Nên thay phần `Github-Guide-Contribute-to-earn` bằng tên repo `rada.content` cho dễ hiểu.